### DISCUSSION TITLE
Pictures of Pensions 🖼️

### DISCUSSION THREAD

**Discussion by declann** [qmd-lab/closeread#154] [Show and tell]
Hi! Just wanted to share my blog post that I made with closeread: https://calcwithdec.dev/posts/pictures-pensions/
The code is on GitLab, [here](https://gitlab.com/declann/calcwithdec.dev/-/blob/main/posts/pictures-pensions/index.qmd?ref_type=heads)

This closeread is OJS heavy, and a reactive [Vega](https://vega.github.io/vega/) visualization is really the star of it.

All the calculations are made with [calculang](https://calculang.dev/), and I'm exploring narrative visualization tools such as closeread because I think this style can be really helpful to communicate about models generally. The abstractions in closeread were really enjoyable to use, and it's awesome to have this functionality and quality output available in Quarto.

**Comment by jimjam-slam** [comment]
Fantastic work, @declann! There's some fantastic stuff going on here — I particularly love the integration of the charts and the inline OJS text. Please do enter it I the [Closeread Prize](http://posit.co/blog/closeread-prize-announcement/) — it's now extended to Jan 5, so no hurry, but you can ofc submit now if you like!

**Reply by declann** [reply]
Thanks James! Sounds good, will add it there soon.

**Comment by axkent** [comment]
Intrigued by the use of 1 inflows chart and having it gently change as the user scrolls. I think that's much more engaging than a conventional article that may have to print several charts to get the same message across.

I'm new to OJS, but I definitely want to learn more about it to more smoothly communicate my datasets. Great work!

**Reply by declann** [reply]
Thanks Axel!
OJS is really great for what you want! Good luck!

**Comment by edenian-prince** [comment]
this is really cool!
